pkg/flow/options.go

```diff
@@ -0,0 +1,27 @@
+package flow
+
+import (
+	"github.com/grafana/agent/pkg/cluster"
+	"github.com/grafana/agent/pkg/flow/logging"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// ModuleOptions holds static options for module.
+type ModuleOptions struct {
+
+	// Logger to use for controller logs and components. A no-op logger will be
+	// created if this is nil.
+	Logger *logging.Logger
```

Comment:
I am a little torn on having this be separate from flow since flow has duplicate values but this feels slightly better referencing flow directly and creating that dependency.